plugins/NavStars/src/NavStars.cpp

```diff
@@ -450,6 +450,10 @@ QString NavStars::getCurrentNavigationalStarsSetDescription() const
 			txt = q_("These 37 stars were used by the Apollo space program to navigate to the Moon from 1969-1972, Apollo 11 through Apollo 17.");
 			break;
 		}
+		case Skywatcher:
+		{
+			txt = q_("This list contains all the alignment stars for the Skywatch SynScan hand controller and SynScan Pro App.");
+		}
```

Comment:
maybe add a `break` also here, even if it's the last case.

Reply:
Yes, not my code. Indeed the braces could go away, would also save a few lines. Can you also clean this up while you are here?

Reply:
> some compilers complain if a switch statement is missing a default

They only complain if the switch parameter is an enum, and there are enum values that are not represented in the cases. By adding a default to an enum that must consider all defined enum values you lose a useful diagnostic.